Adding Managing Redash Groups with OKTA attribute statements section (#627)
getredash/website: src/pages/kb/user-guide/users/authentication-options.md

You can now log in to Redash using Okta SSO.

> ### Managing Redash Groups with OKTA attribute statements

Reviewer: ```suggestion
### Managing Redash Groups with Okta attribute statements
```

> Ops teams often find a need to manage group memberships via OKTA. Follow the below steps inoder to configure okta so that it passes RedashGroups attribute:

Reviewer: ```suggestion
Follow the below steps in order to configure Okta so that it passes the `RedashGroups` attribute:
```

> 1. Goto your OKTA application inside control panel and add **RedashGroups** in SAML settings where you have previously added FirstName and LastName and then save your changes:

Reviewer: ```suggestion
1. To your Okta control panel, add **RedashGroups** in SAML settings where you have previously added FirstName and LastName and then save your changes:
```

| Name         | Value                |
| ------------ | -------------------- |
| FirstName    | user.firstName       |
| LastName     | user.lastName        |
| RedashGroups | appuser.RedashGroups |

{% callout info %}
Name format should be left as **Basic**
{% endcallout %}

2. In the Admin Console, go to **Directory > Profile Editor**, and find the user profile for redash application.

3. In the Attributes screen that opens, click Add Attribute. Add a new attribute with below configuration:

| Name          | Value         |
| ------------- | ------------- |
| Data type     | string array  |
| Variable name | RedashGroups  |
| Scope         | User personal |

4. On the **Applications** page, click the **Assigments** tab. Now you can edit User Assigments and add required RedashGroups for the user.

{% callout info %}

> You can also control attributes on OKTA Group level by removing the Scope from User personal.

Reviewer: ```suggestion
You can also control attributes on the Okta Group level by removing the Scope from User personal.
```
